_posts/2019/2019-06-25-euruko-2019-conference-on-a-ship.md

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
@gersanco please PR that post to `master` (hope my commits will remain then...?)

Reply:
Changed!!